components/Testimonials.js

```diff
@@ -2,7 +2,7 @@ import Link from "./Link";
 
 export default function Testimonials({ data }) {
   return (
-    <div className="bg-white py-24 sm:py-32">
+    <div className="bg-white pt-0 pb-24 sm:pt-0 pb-24"> 
```

Comment:
There is some duplicate and extra white space, try this...

```suggestion
    <div className="bg-white pt-0 pb-24 sm:pt-0">
```

Reply:
It looks like the force push revert your latest changes. I will commit my suggestions

Reply:
> It looks like the force push revert your latest changes. I will commit my suggestions

That means merge commit was better option. Will take care of that in future. Thanks a lot!

Reply:
I very rarely use force push, you made improvements and then in the following commit with a force you removed them

No problem, we are all learning and these things take practice. Next time will be easier, keep going Ekta